test/CodeGenerator/GeneratorTraitTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Di\CodeGenerator;
+
+use Laminas\Di\CodeGenerator\GeneratorTrait;
+use Laminas\Di\Exception\GenerateCodeException;
+use PHPUnit\Framework\TestCase;
+
+final class GeneratorTraitTest extends TestCase
+{
+    public function testEnsureDirectoryGivenInvalidDirectoryNameExpectedErrorMessageContainInvalidDirectoryName(): void
+    {
+        $invalidDir = 'http://www.invalid-directory';
+
+        $this->expectException(GenerateCodeException::class);
+        $this->expectErrorMessage("Could not create output directory: {$invalidDir}");
```

Comment:
Please hardcode or concatenate the value here: string interpolation is a mess to read 😁

Reply:
You just need to send the fix-up commits to the same branch